app/components/subheader/model.js

```diff
@@ -4,6 +4,6 @@ const sanitize = require('../../services/universal/sanitize');
 
 module.exports.save = function (ref, data) {
   data.text = sanitize.validateTagContent(sanitize.toSmartText(data.text || ''));
-
+  data.subheaderid = `${data.subheaderid || data.text}`.trim().replace(/\s+/g, '-');
```

Comment:
Remember that properties should be camelCased, for example
```suggestion
  data.subheaderId = `${data.subheaderId || data.text}`.trim().replace(/\s+/g, '-');
```